packages/cli-test/src/cli/shell.ts

```diff
@@ -252,3 +249,38 @@ export const shell = {
     });
   },
 };
+
+/**
+ * @description Returns arguments used to pass into child_process.spawn or spawnSync. Handles Windows-specifics hacks.
+ */
+function getSpawnArguments(
+  command: string,
+  args: string[],
+  env: ReturnType<typeof shell.assembleShellEnv>,
+  shellOpts?: Partial<child.SpawnOptionsWithoutStdio>,
+): [string, string[], child.SpawnOptionsWithoutStdio] {
+  if (process.platform === 'win32') {
+    // In windows, we actually spawn a command prompt and tell it to invoke the CLI command.
+    // The combination of windows and node's child_process spawning is complicated: on windows, child_process strips quotes from arguments. This makes passing JSON difficult.
+    // As a workaround, by telling Windows Command Prompt (cmd.exe) to execute a command to completion (/c) and leave spaces intact (/c), combined with feeding arguments as an argument array into child_process, we can get around this mess.
+    const windowsArgs = ['/s', '/c'].concat([command]).concat(args);
```

Comment:
This is the root of the Windows compatibility hack

Reply:
It's described in the comment - let me know if you have a suggestion on wording or formatting this differently to make it possibly clearer

Reply:
Oh I see 🤡 `execute a command to completion (/c) and leave spaces intact (/s)`

Reply:
I found it really helpful to see it spelled out that this means Windows will execute `cmd.exe /c /s "slack cli command here"` and would suggest that get added to the comment as well; for a while I thought it was `slack \c \s datastore etc.`, which didn't work when I tried it on my Windows laptop 😅